tests/integration/components/bs-nav-test.js

```diff
@@ -59,6 +59,13 @@ module('Integration | Component | bs-nav', function (hooks) {
     assert.dom('.nav > li').exists({ count: 3 }, 'it has the nav item');
     assert.dom('.nav > li > a[href="/"]').exists({ count: 2 }, 'it has the nav link');
     assert.dom('.nav > li.dropdown').exists({ count: 1 }, 'it has a dropdown as a nav item');
+
+    await click('.nav a.dropdown-toggle');
+    assert.equal(
+      this.element.querySelector('.nav > li.dropdown .dropdown-menu').style.length,
+      0,
+      'Dynamic positioning not applied when inNav is set to true'
+    );
```

Comment:
Can we assert instead that the dropdown is actually in the navbar? Can we compare the `.nav` box position with the `.dropdown-menu` box position and assert that it is inside of it? In that case we would test less the implementation details but more what is expected from end-user's perspective.

Reply:
Good question. Thinking about it more, I don't actually think we can check the position for this test. The in nav behavior happens on both the mobile and non-mobile styles. The non-mobile dropdown looks essentially identical to the dynamically positioned version and is still a "popup", meaning there isn't a difference that can be checked using box positions relative to the menu. I'd be happy to change this further, but I'm not sure how to do it.

Reply:
Thanks for looking into it. Seems like asserting positioning of the element would only work (and be meaningful) for mobile breakpoints. I'm not aware of a strategy how we could set viewport size for a specific test. Likely it wouldn't be worth it as we already have good test coverage asserting against other aspects. Let's keep it as it is. 👍